src/sentry/models/externalissue.py

```diff
@@ -15,6 +15,7 @@ class ExternalIssue(Model):
     date_added = models.DateTimeField(default=timezone.now)
     title = models.TextField(null=True)
     description = models.TextField(null=True)
+    metadata = EncryptedJsonField(null=True)
```

Comment:
This should probably be a plain JSON field instead.

Reply:
Yeah agreed